spring-cloud-gcp-kms/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+	<appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+		<!-- encoders are assigned the type
```

Comment:
Remove comment? This is not relying on default.

Reply:
Why do you say it's not relying on default? When you use `<encoder>` without specifying the class, it's supposed to be the `PatternLayoutEncoder`.

Reply:
I meant that there is an explicit `<pattern>` tag, so it's obvious that `PatternLayoutEncoder` is used even without the comment.